### PR TITLE
Retools the Syndicate Infiltrator (For The Second Time)

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
...Y'ever just get an idea, get inspiration - and just fuckin' go for it?
Anyways, anyways - I've grown a lot less timid since I last touched this shuttle (#65880), been around the block more and started adding tools I didn't have access to, so to celebrate and start the new year: here's round two, mainly to flex how far I've come, even from way before that PR. And also to fix that the former telecomms room was now just essentially "Sit here to charge, EVA fans!" - Hell, I even included a lore terminal, though I'm.. not exactly proud of the writing inside of it. Ah well, someone else can write something better if they're bothered.

Most of these rooms aren't altered, simply redecorated. EVA and Engineering are the obvious exceptions. And yes - Medbay's green. I should probably leave a note to future mappers that since every syndicate cell is different, they don't need to conform to my color choices... ah well.

*So.. what happened to the diorama?*
It's been moved over to the nukie base proper:
image
## Why It's Good For The Game
For being a literal flying terrorist compound, the infiltrator's humorously spacious. This cuts some of the weird use of space out, injects a little bit of the firebase's aesthetic into the thing with the catwalk tile "walls", and's what I feel is as close as you can get to making this thing perfect without altering the hull at all.
(And lose *that* shape? We already lost the airlock being on the Starboard Fore of the thing!)
## Changelog
:cl:
add: The Syndicate Infiltrator's been retooled... AGAIN. There isn't the sheen this time, nor that complimentary taco. Sorry. We got to it first.
spellcheck: Added a period to one of the syndicate keycards. Gotta keep my GBP up somehow.
/:cl:

~~window dressing CL tag when~~

### CODE REVIEW

**Comment by Pumpkinoe** [comment]
pretty cool but the green tinted medbay sign looks a little grody (and the medbay would look better if it kept the clean white tiles)
other that and some weird siding inconsistencies,  this rocks!

**Comment by unit0016** [comment]
I'd actually like to get around to GAGS-ifying department signs as it'd solve this, but've been putting it off because it'd be pathing HELL.
Also felt with the last PR (and still do) that direct white tiles on plastitanium is too harsh a visual difference, so I kept it to mostly regular iron.

**Comment by Pumpkinoe** [comment]
tbh the harsh visual difference is striking and makes sense for something so important that you'd kind of want your eyes to gravitate toward it

**Comment by KittyNoodle** [comment]
can you please add a method of disabling the turrets, sucks to try and rescue a traitor only for them to get ganked by the penetrators

**Comment by unit0016** [comment]
> tbh the harsh visual difference is striking and makes sense for something so important that you'd kind of want your eyes to gravitate toward it

<details>
<summary>Flashbang</summary>
<br>

image

</details>

Definitely not a fan. Too bright for this environ.

**Comment by unit0016** [comment]
> can you please add a method of disabling the turrets, sucks to try and rescue a traitor only for them to get ganked by the penetrators

Turret control panels need a serious refactor before I'd feel comfortable adding this, seeing as I'd either need to sprinkle each turret with the bridge's `/area/` OR have a turret control panel per `/area/` with a turret otherwise, both.. less than ideal solutions.

**Comment by Pumpkinoe** [comment]
alright, fair on the white tiles one
but the herringbone files with 1/4 tile decals on it does not look great

**Comment by SomeoneYouProbablyKnow** [comment]
Feels a bit too busy? with the floor markings and all but it's hard to say from this perspective and im too lazy to see it ingame with proper visibility. But It's not like even nuckies spend much time on the shuttle anyway, guess i trust your expertise.
Side-note : didnt the hallaway to the cockpit have chairs before? or am i tripping?

**Comment by Fikou** [comment]
too busy
image
this is not good, its a logo on a simple background, not everything has to be lit up

**Comment by unit0016** [comment]
> Side-note : didnt the hallaway to the cockpit have chairs before? or am i tripping? It feels like a good place to regroup as you all buckle there.

It did, they've been gone since the prior PR.

And.. Fair, the fore may be overdecorated slightly. I'll deal with that and the decals under the emblem, even if I'm fond of the latter.

File: code/modules/mapfluff/centcom/nuke_ops.dm [hunk 1]
```diff
@@ -36,10 +36,27 @@
...
+	content = list(
+		"<b>C:/Users/ADMIN/Desktop/'Chords, Read Me.txt'</b><br> \
+		<br> \
+		Chords, I'm sorry about our last night out. I know you didn't like what you saw, and I know I dissapointed you, \
+		greatly, when I acted the way I did - did what I did. I'm sorry, Chords, I can't hammer that home enough. \
+		I've spent days now trying to think of something to say, trying to think of some way to fix things, and I... \
+		I can't find any other than to sit here, in this shitty, dingy ass maintenance corridor, and just... type. \
+		Please. I'll remember to put on my oxygen mask AND run internals properly next operation. Talk to me again.<br> \
+		<br> \
+		> sudo rm -rf") // This one goes out to every flukie who's forgotten their internals, including myself. Godspeed you dumb, dumb dweebs.
```
Comment (Mothblocks) [inline code/modules/mapfluff/centcom/nuke_ops.dm:62]: in before loser nerds make pr's to change either this or the windows file path name for inconsistencies

Comment (unit0016) [reply code/modules/mapfluff/centcom/nuke_ops.dm:62]: Unintentional but incredibly funny, dork bait

**Comment by unit0016** [comment]
> (also im a panorama hater this should be on the syndie base not the transport shuttle but it was there before so shrug)

I could move it over, unsure what I'd put in the main hall in replacement though.

**Comment by unit0016** [comment]
Okay, toned down the decals heavily, replaced the long-gone chairs with benches.

File: code/modules/mapfluff/centcom/nuke_ops.dm (hunk 1 above)
Comment (Mothblocks) [reply code/modules/mapfluff/centcom/nuke_ops.dm:62]: yeah let it be known they shall all be closed

**Comment by Fikou** [comment]
> > (also im a panorama hater this should be on the syndie base not the transport shuttle but it was there before so shrug)
> 
> I tried moving it over. This is the only 3x3 spot that'd fit it: image
> 
> Not exactly _great_, on top of the fact the best way I could figure out to replace the lost space and decal was to... make it look like a baseball. Think it's stayin' on the shuttle.

OH i meant diorama the little carp thingy sorry im stupid

**Comment by unit0016** [comment]
OH PFFTTTT. No I can move that easy-peasy, I know a good spot for it.

**Pull request closed without merging**